src/main/java/pl/wojciechkostecki/revel/model/User.java

```diff
@@ -0,0 +1,28 @@
+package pl.wojciechkostecki.revel.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Data
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
```

Comment:
why do we even need this? It doesnt add anything - column will be named `id` by default